.dockerignore

```diff
@@ -0,0 +1,14 @@
+libbuild/
```

Comment:
Without this file, some caches built on local computer will be included in the image build environment, and then the docker build will fail because of cache inconsistency.

Reply:
Yeah, we were really missing a dockerignore file. Thank you so much for creating one